pyproject.toml

```diff
@@ -1,39 +1,55 @@
+[build-system]
+requires = [
+    "setuptools>=64",
+    "setuptools-scm[toml]>=6.2"
+]
+build-backend = "setuptools.build_meta"
+
 
 [project]
 name = "carbonplan-project"
```

Comment:
Does putting any field that the repo user has to change in <> help? We could also detail all the fields that should be updated in the README.md
ex: name = "<project-name>"